tests/command.dev.test.js

```diff
@@ -14,6 +15,9 @@ const { withDevServer } = require('./utils/dev-server')
 const { startExternalServer } = require('./utils/external-server')
 const { withSiteBuilder } = require('./utils/site-builder')
 
+const CONCURRENCY = 1
+const semaphore = process.env.CI === 'true' ? new Semaphore(CONCURRENCY) : null
```

Comment:
Would the following work too?

```js
const testFunc = process.env.CI === 'true' ? test.serial.bind(test) : test

testFunct('test title', (t) => { ... })
```

Reply:
You're right, I was playing with different values or `CONCURRENCY` (`4->2->1`).
We can remove the dependency and enforce `serial`.

Reply:
I chose to do it a bit different. It required disabling a lint rule, but makes the spec consistent with other ones.